examples/platforms/cc2650/random.c

```diff
@@ -145,6 +142,11 @@ ThreadError otPlatRandomSecureGet(uint16_t aInputLength, uint8_t *aOutput, uint1
  */
 int mbedtls_hardware_poll(void *data, unsigned char *aOutput, size_t aLen, size_t *oLen)
 {
+    ThreadError error;
+
     (void)data;
-    return TRNGPoll(aOutput, aLen, oLen);
+    error = TRNGPoll(aOutput, aLen);
```

Comment:
Seems that `TRNGPoll` always return 0, can we return `TRNGPoll()` directly after `*oLen = aLen;`? `error` seems to be a little redundant.